test/work_item_repository.go

```diff
@@ -22,32 +22,34 @@ type WorkItemRepository struct {
 		result1 *app.WorkItem
```

Comment:
I agree with you: I had to use counterfeiter to regenerate that file a couple of times already. AFAICS, it's only used in `controller/workitem_blackbox_test.go`, `controller/workitem_test.go`and `test/db.go`, but I think that we should be able to get rid of it now (in another PR, though)